packages/block-editor/src/hooks/font-size.js

```diff
@@ -186,13 +186,18 @@ const withFontSizeInlineStyles = createHigherOrderComponent(
 				style?.typography?.fontSize
 			).size;
 
-			props.wrapperProps = {
-				...wrapperProps,
-				style: {
-					fontSize: fontSizeValue,
-					...wrapperProps?.style,
+			const newProps = {
+				...props,
+				wrapperProps: {
+					...wrapperProps,
+					style: {
+						fontSize: fontSizeValue,
+						...wrapperProps?.style,
+					},
 				},
 			};
+
+			return <BlockListBlock { ...newProps } />;
```

Comment:
I wonder if returning here will cause "remounting" of the component? Can we instead keep the mutation here but declarer newProps at the top of the component (and spread props)

Reply:
Done in ae0ba69!